config/schedule.rb

```diff
@@ -6,12 +6,12 @@
 app_config = YAML.load_file(File.join(__dir__, 'application.yml'))
 
 env "MAILTO", app_config["SCHEDULE_NOTIFICATIONS"] if app_config["SCHEDULE_NOTIFICATIONS"]
+set :path, '/home/openfoodnetwork/apps/openfoodnetwork/current'
```

Comment:
I dont think we can have this path in ofn. we need to move this to ofn-install in some way, right?

Reply:
Updated

Reply:
Associated PR here: https://github.com/openfoodfoundation/ofn-install/pull/631

Reply:
:ok_hand: